Managed_SCCA_Broker_(SCCAv2)/DESTROY.md

```diff
@@ -0,0 +1,71 @@
+# Guide to Destroy SCCAv2 Landing Zone Using destroy_lz.py
+
+### Prerequisites:
+1. Ensure all four sub-compartments (VDMS, VDSS, Logging, and Config) are present in the parent compartment
+2. Do not run terraform destroy before running the destroy_lz.py script
+   1. If you have already run terraform destroy, navigate to the vault resource in the cloud console and manually set the status to "active"
+
+### 1.	Navigate to the destroy_lz.py script:
+     /Managed_SCCA_Broker_(SCCAv2)/destroy_lz.py
+The script is in the same directory as this guide, titled "destroy_lz.py"
+
+### 2.	Download and move this script into your local landing zone directory:
+For example: /User/mylandingzone/scca_cis_multi_tenancy/destroy_lz.py
+
+### 3.	Run the destroy_lz.py script:
+Run this command in the command line (replace the information in the brackets with your compartment's information):
+
+```text
+python3 destroy_lz.py -c [YOUR LANDING ZONE COMPARTMENT NAME] -r [YOUR REGION KEY] -l [YOUR RESOURCE LABEL]  --template_name [TEMPLATE NAME (options: CHILD, PARENT, or WORKLOAD]
+```
+For example: python3 destroy_lz.py -c OCI-SCCA-LZ-CHILD-Home-RIC-miachildtest -r RIC -l miachildtest --template_name CHILD
```

Comment:
Do we want to use a more generic example name ("exampletenancy") instead of "miachildtest"